AppSyncRealTimeClient.xcodeproj/xcshareddata/xcschemes/AppSyncRealTimeClient.xcscheme

```diff
@@ -45,6 +45,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      enableThreadSanitizer = "YES"
```

Comment:
Is this enabling TSan by default when running in DEBUG, and if so, is that what we want?